pkg/controlplane/instance.go

```diff
@@ -748,6 +749,7 @@ var (
 		rbacv1.SchemeGroupVersion,
 		storageapiv1.SchemeGroupVersion,
 		schedulingapiv1.SchemeGroupVersion,
+		flowcontrolv1.SchemeGroupVersion,
 	}
 
 	// legacyBetaEnabledByDefaultResources is the list of beta resources we enable.  You may only add to this list
```

Comment:
we still need to keep v1beta3 enabled here, right?

https://github.com/kubernetes/kubernetes/pull/121089/commits/f863e7007d50d884635662c1c45f0e9d0f3c94ba#diff-05cd394e18da9e91dbf8b3d82e1e0813be670ee37db43ca5af6f91a39f78d08fL758-L759

```
	legacyBetaEnabledByDefaultResources = []schema.GroupVersionResource{
		flowcontrolv1beta3.SchemeGroupVersion.WithResource("flowschemas"),                 // deprecate in 1.29, remove in 1.32
		flowcontrolv1beta3.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // deprecate in 1.29, remove in 1.32
	}
```

Reply:
Makes sense to me.  See also the comment already in there, "deprecate in 1.29, remove in 1.32".

BTW, I _also_ see the v1beta3 in `betaAPIGroupVersionsDisabledByDefault`; is that right?